Patch adapted for PEAR current files.

Original patch:

From 0c4079ffab0ff669772ee39ce599e4699ddde32f Mon Sep 17 00:00:00 2001
From: Remi Collet <fedora@famillecollet.com>
Date: Sun, 18 May 2014 09:22:39 +0200
Subject: [PATCH] Cleanup some license headers Fix pear bug #20274 Also see
 21ce607014ae3445147be2e52ceacdca0ca7b503

---
 Text/Wiki/Render/Cowiki/Prefilter.php | 45 ++++++++++++-----------------------
 Text/Wiki/Render/Doku/Prefilter.php   | 45 ++++++++++++-----------------------
 Text/Wiki/Render/Latex/Prefilter.php  | 34 +-------------------------
 Text/Wiki/Render/Plain/Prefilter.php  |  2 +-
 Text/Wiki/Render/Tiki/Prefilter.php   | 45 ++++++++++++-----------------------
 5 files changed, 47 insertions(+), 124 deletions(-)

diff --git a/Text/Wiki/Render/Latex/Prefilter.php b/Text/Wiki/Render/Latex/Prefilter.php
index 974697a..97fa666 100644
--- a/Text/Wiki/Render/Latex/Prefilter.php
+++ b/Text/Wiki/Render/Latex/Prefilter.php
@@ -1,7 +1,7 @@
 <?php
 // vim: set expandtab tabstop=4 shiftwidth=4 softtabstop=4:
 /**
- * This class implements a Text_Wiki_Render_Xhtml to "pre-filter" source text so
+ * This class implements a Text_Wiki_Render_Latex to "pre-filter" source text so
  * that line endings are consistently \n, lines ending in a backslash \
  * are concatenated with the next line, and tabs are converted to spaces.
  *
@@ -15,38 +15,6 @@
  * @link       http://pear.php.net/package/Text_Wiki
  */
 
-/* vim: set expandtab tabstop=4 shiftwidth=4: */
-// +----------------------------------------------------------------------+
-// | PHP version 4                                                        |
-// +----------------------------------------------------------------------+
-// | Copyright (c) 1997-2003 The PHP Group                                |
-// +----------------------------------------------------------------------+
-// | This source file is subject to version 2.0 of the PHP license,       |
-// | that is bundled with this package in the file LICENSE, and is        |
-// | available through the world-wide-web at                              |
-// | http://www.php.net/license/2_02.txt.                                 |
-// | If you did not receive a copy of the PHP license and are unable to   |
-// | obtain it through the world-wide-web, please send a note to          |
-// | license@php.net so we can mail you a copy immediately.               |
-// +----------------------------------------------------------------------+
-// | Authors:                            |
-// +----------------------------------------------------------------------+
-//
-// $Id: Prefilter.php 248435 2007-12-17 16:19:44Z justinpatrin $
-
-
-/**
-* 
-* This class implements a Text_Wiki_Render_Latex to "pre-filter" source text so
-* that line endings are consistently \n, lines ending in a backslash \
-* are concatenated with the next line, and tabs are converted to spaces.
-*
-* @author Jeremy Cowgar <jeremy@cowgar.com>
-*
-* @package Text_Wiki
-*
-*/
-
 class Text_Wiki_Render_Latex_Prefilter extends Text_Wiki_Render {
     function token()
     {
diff --git a/Text/Wiki/Render/Plain/Prefilter.php b/Text/Wiki/Render/Plain/Prefilter.php
index 17d48eb..1879fd1 100755
--- a/Text/Wiki/Render/Plain/Prefilter.php
+++ b/Text/Wiki/Render/Plain/Prefilter.php
@@ -1,7 +1,7 @@
 <?php
 // vim: set expandtab tabstop=4 shiftwidth=4 softtabstop=4:
 /**
- * This class implements a Text_Wiki_Render_Xhtml to "pre-filter" source text so
+ * This class implements a Text_Wiki_Render_Plain to "pre-filter" source text so
  * that line endings are consistently \n, lines ending in a backslash \
  * are concatenated with the next line, and tabs are converted to spaces.
  *
-- 
1.9.3

